_submissions/lua_script/lua-command+.md

```diff
@@ -0,0 +1,34 @@
+# Script info
+
+- Name: lua-command+
+- Author: iThorgrim-hub
+- Repository: https://github.com/iThorgrim-hub/Eluna-Catalogue/tree/lua-command+
+- Download: https://github.com/iThorgrim-hub/Eluna-Catalogue/archive/command+_v1.0.zip

```

Comment:
A little suggestion: change the link to the release page.

https://github.com/iThorgrim-hub/Eluna-Catalogue/releases

Reply:
No problem ;)

Reply:
It' done